econml/utilities.py

```diff
@@ -235,14 +253,13 @@ def cross_product(*XS):
     X2 : n x d2 matrix
         Second matrix of n samples of d2 features
         (or an n-element vector, which will be treated as an n x 1 matrix)
-    …
 
     Returns
     -------
-    A : n x (d1*d2*...) matrix
+    n x (d1*d2*...) matrix
```

Comment:
Add the name for the output back so that the docstring is rendered better
```suggestion
    A : n x (d1*d2*...) matrix
```

Reply:
Done